src/supertux/sector.hpp

```diff
@@ -145,27 +146,20 @@ class Sector
                          MovingObject &object) const;
 
   /**
-   * Checks if the specified rectangle is free of (solid) tiles.
-   * Note that this does not include static objects, e.g. bonus blocks.
+   * Wrapper for SectorCollisionDetector's is_free_of_tiles function.
```

Comment:
I don't know if it is a good idea to completely delete this comment (@maxteufel / @tobbi)?

Reply:
The full comment is still here: https://github.com/SuperTux/supertux/pull/866/files#diff-3286e434910f100eed55179300cff945R45